src/View/AbstractDirectoryView.vala

```diff
@@ -282,12 +285,35 @@ namespace Files {
         public signal void path_change_request (GLib.File location, Files.OpenFlag flag, bool new_root);
         public signal void selection_changed (GLib.List<Files.File> gof_file);
 
+        static construct {
```

Comment:
You can just use Gtk.Bin here there's Granite.Bin in Gtk4 as a drop-in replacement